controllers/operands/deploymentHandler.go

```diff
@@ -15,23 +15,41 @@ import (
 	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/common"
 )
 
+type deploymentOperand struct {
+	operand *genericOperand
+}
+
+func (dp deploymentOperand) ensure(req *common.HcoRequest) *EnsureResult {
+	return dp.operand.ensure(req)
+}
```

Comment:
why?

Reply:
in the specific handlers for UI plugin and UI proxy, we want to convert `deploymentOperand` to `[]Operand`, so it can be added to the operands list in `operandHandler.go` via GetHandler() function.
in order for it to implement `Operand`, it must have the `ensure()` method.